src/components/StatueMenuContainer.js

```diff
@@ -5,9 +5,11 @@ import style from "./StatueMenuContainer.module.css";
 export class StatueMenuContainer extends Component {
   render() {
     return (
-      <div className={style.container}>
-        <h1>Statues</h1>
-        <StatueMenu />
+      <div className="col-lg-3 col-md-6 col-sm-12">
+        <div className={style.container}>
```

Comment:
applying height 100% to this will stretch the red column down to the bottom of the slaver description like the original mockup